### PR TITLE
Fix for compilation errors on SLES and RHEL8

### PR DESCRIPTION
**Pull request by ashwathraop**
#### Describe Bug or Feature

Jenkins build fails on certain platforms because of signature mismatch in pbs_sched.c

#### Describe Your Change

Correct the signature.

### CODE REVIEW

**Comment by ashwathraop** [comment]
Machine | OS | Build_Status | Artifacts link | PTL_REPORT | Total_Tests | Passed | Skipped | Failed | Error | Timedout | Build_Start_Time[PDT] | Test_Time | Total_Build_Time
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
pbi-04-r7 |   | Fail | NA | NA | NA | NA | NA | NA | NA | NA | NA | NA | NA
pbi-18-s12 | linux312_x86_64 | Success | Artifacts | NA | 1297 | 1286 | 11 | 0 | 0 | 0 | 10:44:18 | 1:32:44 | 01:20:12
pbi-26-s12-cray | linux312_x86_64 | Success | Artifacts | NA | 1297 | 1285 | 12 | 0 | 0 | 0 | 10:11:26 | 1:29:41 | 01:12:24
pbi-31-u16 | linux413_x86_64 | Success | Artifacts | NA | 1298 | 1286 | 11 | 0 | 1 | 0 | 11:55:10 | 2:00:59 | 01:48:01
pbi-32-s12 | linux312_x86_64 | Success | Artifacts | NA | 1297 | 1284 | 11 | 0 | 2 | 0 | 09:42:19 | 1:35:50 | 00:40:53
pbi-33-r7 | linux310_x86_64 | Success | Artifacts | NA | 1297 | 1284 | 11 | 0 | 2 | 0 | 09:42:09 | 2:13:07 | 00:34:39
pbi-34-s15 | linux412_x86_64 | Success | Artifacts | NA | 1298 | 1283 | 11 | 1 | 3 | 0 | 11:14:11 | 3:02:11 | 01:11:23
pbi-35-u18 | linux415_x86_64 | Success | Artifacts | NA | 1298 | 1286 | 11 | 0 | 1 | 0 | 11:41:58 | 2:07:17 | 01:42:28
pbi-37-s15-cray | linux412_x86_64 | Success | Artifacts | NA | 1298 | 1282 | 12 | 1 | 3 | 0 | 09:40:54 | 3:00:32 | 00:53:15
pbi-39-r8 | linux418_x86_64 | Fail | Artifacts | NA | NA | NA | NA | NA | NA | NA | 09:40:54 | NA | 00:19:57
pbi-xarm-01-s15 | linux412_aarch64 | Success | Artifacts | NA | 1297 | 1286 | 11 | 0 | 0 | 0 | 09:40:54 | 2:22:20 | 00:56:26
pbi-xarm-02-s15-cray | linux412_aarch64 | Success | Artifacts | NA | 1297 | 1285 | 12 | 0 | 0 | 0 | 09:27:23 | 2:23:58 | 01:03:04
pbi-xarm-03-r8 | linux418_aarch64 | Fail | Artifacts | NA | NA | NA | NA | NA | NA | NA | 10:10:21 | NA | 00:27:12
pbi-xarm-04-u18 | linux415_aarch64 | Success | Artifacts | NA | 1297 | 1286 | 11 | 0 | 0 | 0 | 11:39:42 | 1:32:41 | 01:41:49
pbi-38-2k16 | windows | Fail | Artifacts |   |   |   |   |   |   |   | 09:40:58 |   | 00:28:00

File: src/server/pbsd_main.c [hunk 1]
```diff
@@ -823,23 +823,21 @@ main(int argc, char **argv)
...
 	self.port = pbs_conf.batch_service_port;
 	if (get_max_servers() > 1) {
-		char buf[PBS_MAXHOSTNAME+8];
 		if ((myindex = get_svr_index(&self)) == -1) {
 			fprintf(stderr, "pbsconf error: Wrong Multi Server configuration\n");
 			return 1;
 		}
-		snprintf(buf, sizeof(buf), "%s_%d", daemonname, pbs_conf.batch_service_port);
-		strcpy(daemonname, buf);
+		snprintf(daemonname, (PBS_MAXHOSTNAME + 13), "Server@%s_%d", server_host, pbs_conf.batch_service_port);
```
Comment (hirenvadalia) [inline src/server/pbsd_main.c:836]: instead of `(PBS_MAXHOSTNAME + 13)` use `sizeof(daemonname) - 1`

**Comment by ashwathraop** [comment]
Latest build report:

Machine | OS | Build_Status | Artifacts link | Build_Start_Time[PDT] | Total_Build_Time
-- | -- | -- | -- | -- | --
pbi-04-r7 | linux310_x86_64 | Success | Artifacts | 17:38:23 | 00:56:37
pbi-18-s12 | linux312_x86_64 | Success | Artifacts | 17:38:24 | 01:26:07
pbi-26-s12-cray | linux312_x86_64 | Success | Artifacts | 17:38:35 | 01:14:04
pbi-31-u16 | linux413_x86_64 | Success | Artifacts | 17:38:32 | 01:42:00
pbi-32-s12 | linux312_x86_64 | Success | Artifacts | 17:40:06 | 00:40:32
pbi-33-r7 | linux310_x86_64 | Success | Artifacts | 17:39:56 | 00:35:37
pbi-34-s15 | linux412_x86_64 | Success | Artifacts | 17:38:34 | 01:16:24
pbi-35-u18 | linux415_x86_64 | Success | Artifacts | 17:38:34 | 01:46:57
pbi-37-s15-cray | linux412_x86_64 | Success | Artifacts | 17:38:35 | 00:56:58
pbi-39-r8 | linux418_x86_64 | Success | Artifacts | 17:38:35 | 01:01:17
pbi-xarm-01-s15 | linux412_aarch64 | Success | Artifacts | 17:38:35 | 00:56:38
pbi-xarm-02-s15-cray | linux412_aarch64 | Success | Artifacts | 17:24:23 | 01:02:01
pbi-xarm-03-r8 | linux418_aarch64 | Success | Artifacts | 17:38:35 | 01:14:36
pbi-xarm-04-u18 | linux415_aarch64 | Success | Artifacts | 17:38:36 | 01:45:55
pbi-38-2k16 | windows | Fail | Artifacts | 17:38:08 | 00:28:15

Windows build failed looking for pbspro.spec:
FINDSTR: Cannot open C:\Users\pbsbuild\builds\workspace\PBSPro_BUILD_NODES\pbspro\pbspro.spec
I am not sure if its Jenkins issue ot=r we have to update some part of our code.

**Comment by ashwathraop** [comment]
Results after rebase:

Machine | OS | Build_Status | Artifacts link | Build_Start_Time[PDT] | Total_Build_Time
-- | -- | -- | -- | -- | --
pbi-04-r7 | linux310_x86_64 | Success | Artifacts | 06:49:39 | 01:01:56
pbi-38-2k16 | windows | Success | Artifacts | 06:50:05 | 00:37:55

**Comment by hirenvadalia** [comment]
@ashwathraop still travis is failing, so I can't merge as travis is required check, but if those failures are expected then @subhasisb you have to do force merge with admin power 😄

**Pull request merged**